parser/parser_test.go

```diff
@@ -42,7 +42,7 @@ func (s *testParserSuite) TestSimple(c *C) {
 		"max_rows", "min_rows", "national", "row", "quarter", "escape", "grants", "status", "fields", "triggers",
 		"delay_key_write", "isolation", "repeatable", "committed", "uncommitted", "only", "serializable", "level",
 		"curtime", "variables", "dayname", "version", "btree", "hash", "row_format", "dynamic", "fixed", "compressed",
-		"compact", "redundant",
+		"compact", "redundant", "sql_no_cache sql_no_cache", "sql_cache sql_cache",
```

Comment:
It will be composed to "select sql_no_cache sql_no_cache;".The first is sql_cache option, the second is select field.